components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -182,7 +182,7 @@ public OAuth2ClientApplicationDTO findOAuthConsumerIfTokenIsValid(OAuth2TokenVal
         responseDTO.setExpiryTime(getAccessTokenExpirationTime(accessTokenDO));
 
         // Adding the AccessTokenDO as a context property for further use
-        messageContext.addProperty("AccessTokenDO", accessTokenDO);
+        messageContext.addProperty(OAuthConstants.ACCESS_TOKEN_DO, accessTokenDO);
```

Comment:
Does this add the new claim to the introspection API call response?  I think it needs to be added to the `buildIntrospectionResponse` method